src/crimson/tools/CMakeLists.txt

```diff
@@ -15,4 +15,4 @@ add_executable(perf-async-msgr perf_async_msgr.cc)
 target_link_libraries(perf-async-msgr ceph-common global ${ALLOC_LIBS})
 
 add_executable(perf-staged-fltree perf_staged_fltree.cc)
-target_link_libraries(perf-staged-fltree crimson-seastore)
+target_link_libraries(perf-staged-fltree crimson-seastore crimson::gtest)
```

Comment:
@myoungwon

I applied this patch in my environment. With it, cmake failed with below error.
(Also I had given the option -DWITH_TESTS=OFF when doing cmake.)

_====================
CMake Error at cmake/modules/BuildBoost.cmake:301 (add_executable):
Target "perf-staged-fltree" links to target "crimson::gtest" but the target
was not found. Perhaps a find_package() call is missing for an IMPORTED
target, or an ALIAS target is missing?
===============================_

I changed crimson::gtest ==> gtest, then I did not see the cmake error or the original error that I mentioned in previous post.

Reply:
If we disable WITH_TESTS, probably, I think gtests is also excluded from the build, right?

Reply:
Hmm, probably we can specialize `EphemeralTestState` not to depend on gtest (including the inheritage to `::testing::WithParamInterface` if tests are disabled (see the usage around `UNIT_TESTS_BUILT`).

Reply:
@cyx1231st @aravind-wdc Updated.